packages/friends/yml/en/john-banks.yml

```diff
@@ -2,15 +2,15 @@ name: John Banks
 slug: john-banks
 gender: male
 description:
-  John Banks (1637-1710) was another faithful and influential minister among the first
-  generation of Quakers. "After it had pleased the eternal, wise God to open his
-  understanding, and to let him see his own state and condition, and reveal his Son in
-  him, he was made willing to give up freely to the heavenly and inward appearance of
-  Christ Jesus, the hope of glory. And as he was obedient thereunto, he was entrusted with
-  a large gift of the ministry, in which he grew, and was made powerful in it, to the
-  turning of many unto the right way of the Lord who were convinced of the evil of their
-  ways, and turned unto Jesus Christ, their free Teacher, and were made to bless the Lord
-  on his behalf." (Testimony of intimate friend, John Bousted)
+  John Banks (1637-1710) was faithful and influential minister among the first generation
+  of Quakers. "After it had pleased the eternal, wise God to open his understanding, and
+  to let him see his own state and condition, and reveal his Son in him, he was made
+  willing to give up freely to the heavenly and inward appearance of Christ Jesus, the
+  hope of glory. And as he was obedient thereunto, he was entrusted with a large gift of
+  the ministry, in which he grew, and was made powerful in it, to the turning of many unto
+  the right way of the Lord who were convinced of the evil of their ways, and turned unto
+  Jesus Christ, their free Teacher, and were made to bless the Lord on his behalf."
+  (Testimony of intimate friend, John Bousted)
```

Comment:
Maybe would it be better to remove the parentheses, and change it to say:

Upon his death, an intimate friend wrote of him: "After it had pleased...

Better? Worse? I'm OK either way...

Reply:
I like your idea better. I may rewrite this one entirely some day.  I think I just used this quote at first because I couldn't come up with anything.